unlock-app/src/__tests__/pages/pages.test.js

```diff
@@ -100,4 +102,63 @@ describe('Pages', () => {
       expect(pageTitle).toBeCalledWith('Dashboard')
     })
   })
+
+  describe('Blog', () => {
+    it('should render title correctly', () => {
+      expect.assertions(1)
+      rtl.render(
+        <Provider store={store}>
+          <Blog posts={[]} />
+        </Provider>
+      )
+
+      expect(pageTitle).toBeCalledWith('Blog')
+    })
+    it('should render post preview correctly', () => {
+      expect.assertions(3)
+
+      const posts = [
+        {
+          slug: 'test1',
+          title: 'Test post',
+          publishDate: 'November 23, 1963',
+          authorName: 'Susan Foreman',
+          description: 'One day, I will come back.',
+        },
+      ]
+
+      const page = rtl.render(
+        <Provider store={store}>
+          <Blog posts={posts} />
+        </Provider>
+      )
+
+      expect(page.queryByText(posts[0].title)).not.toBeNull()
```

Comment:
little tip I've learned: if you use `getByText` for anything you expect to be there, if it isn't, jest will print a beautifully formatted html printout of the actual source tree that was generated